specs/Modal.events.spec.js

```diff
@@ -236,9 +237,40 @@ export default () => {
       withModal(props, null, modal => {
         mouseDownAt(mcontent(modal));
         mouseUpAt(moverlay(modal));
+        clickAt(moverlay(modal));
```

Comment:
Same here.

Reply:
Ah now I remember it...

The `mouse down and up` are to test the scenario where you click and hold on a content and release (mouse up) on the overlay.

Reply:
Yes that's it!
It just didn't work when I ran the tests, so I compared it to real behavior and suggesting a way to fix the tests. At least now they works for me :)

Reply:
I guess in the end the result is correct, but I'm not sure if the test "examplifies" the behavior.

I need to think about it.

Reply:
This one too.

If I press down on the content, drag to the overlay, it must not close the modal.

Reply:
```suggestion
```